python/eos/tasks.py

```diff
@@ -48,10 +48,22 @@ def __exit__(self, type, value, traceback):
 
 _tasks = {}
 
-def task(name, output, mode=lambda **kwargs: 'w'):
+def task(name, output, mode=lambda **kwargs: 'w', modules=[]):
```

Comment:
This can be applied to pypmc, dynesty,... right?

Reply:
Correct. ``modules`` can contain a list of string, where each string corresponds to the name of a potential python module. Each module will then be imported and a consistent error message thrown if it is unavailable.

Reply:
Then we should certainly do it (maybe in another PR).

Reply:
Yes. This requires some work though. Currently, those modules are globally imported; those imports now need to be moved into the tasks / classes. I'll open an issue.